subprojects/packagefiles/glfw/examples/meson.build

```diff
@@ -0,0 +1,37 @@
+###  GLFW examples
+#@ Build check
+if not get_option('build-examples')
+  subdir_done()
+endif
+
+#@ Constants
+ex_include = include_directories('../include', '../src', '../deps')
+glad_gl = '../deps/glad_gl.c'
+getopt = '../deps/getopt.c'
+tinycthread = '../deps/tinycthread.c'
+
+#@ Flags
+if is_posix
+  c_flags = ['-D_XOPEN_SOURCE=700']
+endif
```

Comment:
Is this actually being used by anything?

Reply:
Yes! Several of the examples and tests will fail to build without it.

Reply:
No, I mean, when is this variable ever used by passing it as a kwarg to a function?

Reply:
Oops! Fixed.

You pointed out that tinycthread should set "-D_XOPEN_SOURCE=500", but that doesn't seem to work. =\